components/contentful-blog-post-list/contentful-blog-post-list.makeswift.tsx

```diff
@@ -14,7 +14,7 @@ export const props = {
     label: 'Tag',
     async getOptions(query) {
       const response = await fetch('/api/contentful/tags');
-      const body = await response.json();
+      const body: unknown = await response.json();
```

Comment:
Was there a eslint warning for this?

Reply:
`..json()` returns `any`, so if you type body as `unknown` it should work as is. There is also a package that does this implicitly for us: https://www.npmjs.com/package/@total-typescript/ts-reset